operators/multiclusterobservability/controllers/multiclusterobservability/multiclusterobservability_controller.go

```diff
@@ -398,6 +399,12 @@ func (r *MultiClusterObservabilityReconciler) Reconcile(ctx context.Context, req
 		}
 	}
 
+	// create rightsizing component
+	result, err = analyticsctrl.CreateRightSizingComponent(r.Client, r.Scheme, instance, r.Manager)
```

Comment:
You can pass the `ctx` here and use it in the new functions.

Reply:
Now using `ctx`.